apps/crn-frontend/src/network/working-groups/WorkingGroupProfile.tsx

```diff
@@ -13,24 +13,9 @@ const WorkingGroupProfile: FC = () => {
   const workingGroup = useWorkingGroupById(workingGroupId);
 
   if (workingGroup) {
-    const props: ComponentProps<typeof WorkingGroupPage> = {
```

Comment:
This is better mocked in the API as it currently doesn't meet the entire contract. I assume these deficiencies are covered in some ticket